github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -714,6 +715,13 @@ orgs:
           - isaacdorfman
         repos:
           terraform-provider-kubevirt: admin
+      rerun-jobs-hco:
+        description: can rerun failed job in the hyperconverged-cluster-operator repository
+        maintainers:
+        - nunnatsa
+        - tiraboschi
+        repos:
+          hyperconverged-cluster-operator: admin
```

Comment:
This line would make you repo admin. In general we rather strive to remove those privileges from users and rely on peribolos for org and repo management instead. Peribolos will perform the changes from this PR once it gets merged. And it will revert any changes you do to the repo manually.

If you don't need this right, please remove the full repos: section here.